### PR TITLE
fix(select/contextselector): update name for menu search

### LINKED ISSUE #2438: Select/Context selector - update search box element name
let's use `.pf-c-context-selector__menu-search` to match the app launcher.

Also update the checkbox select search from https://github.com/patternfly/patternfly-next/pull/2693/files

### PR DESCRIPTION
**Pull request by christiemolloy**
closes #2438 

## Breaking Changes
* changes `pf-c-context-selector__menu-input` to `pf-c-context-selector__menu-search`
* changes `pf-c-select__menu-input` to `pf-c-select__menu-search`
* changes `--pf-c-context-selector__menu-input--PaddingTop` to `--pf-c-context-selector__menu-search--PaddingTop`
* changes `--pf-c-context-selector__menu-input--PaddingRight` to `--pf-c-context-selector__menu-search--PaddingRight`
* changes `--pf-c-context-selector__menu-input--PaddingBottom` to `--pf-c-context-selector__menu-search--PaddingBottom`
* changes `--pf-c-context-selector__menu-input--PaddingLeft` to `--pf-c-context-selector__menu-search--PaddingLeft`
*  changes `--pf-c-context-selector__menu-input--BottomBorderColor` to `--pf-c-context-selector__menu-search--BorderBottomColor`
* changes `--pf-c-context-selector__menu-input--BottomBorderWidth` to `--pf-c-context-selector__menu-search--BorderBottomWidth`
* changes `--pf-c-select__menu-input--PaddingTop` to `--pf-c-select__menu-search--PaddingTop`
* changes `--pf-c-select__menu-input--PaddingRight` to `--pf-c-select__menu-search--PaddingRight`
* changes `--pf-c-select__menu-input--PaddingBottom` to `--pf-c-select__menu-search--PaddingBottom`
* changes `--pf-c-select__menu-input--PaddingLeft` to `--pf-c-select__menu-search--PaddingLeft`

### CODE REVIEW

**Comment by mcoker** [comment]
@christiemolloy can you also make note that all of the `--pf-c-select__menu-input*` and `--pf-c-context-selector__menu-input*` vars changed to `--pf-c-select__menu-search*` and `--pf-c-context-selector__menu-search*`?

File: src/patternfly/components/ContextSelector/context-selector.scss [hunk 1]
```diff
@@ -37,12 +37,12 @@
...
-  --pf-c-context-selector__menu-input--PaddingRight: var(--pf-global--spacer--md);
-  --pf-c-context-selector__menu-input--PaddingBottom: var(--pf-global--spacer--md);
-  --pf-c-context-selector__menu-input--PaddingLeft: var(--pf-global--spacer--md);
-  --pf-c-context-selector__menu-input--BottomBorderColor: var(--pf-global--BorderColor--100);
-  --pf-c-context-selector__menu-input--BottomBorderWidth: var(--pf-global--BorderWidth--sm);
+  --pf-c-context-selector__menu-search--PaddingTop: var(--pf-global--spacer--sm);
+  --pf-c-context-selector__menu-search--PaddingRight: var(--pf-global--spacer--md);
+  --pf-c-context-selector__menu-search--PaddingBottom: var(--pf-global--spacer--md);
+  --pf-c-context-selector__menu-search--PaddingLeft: var(--pf-global--spacer--md);
+  --pf-c-context-selector__menu-search--BottomBorderColor: var(--pf-global--BorderColor--100);
```
Comment (mcoker) [inline src/patternfly/components/ContextSelector/context-selector.scss]: Nit - in this PR can we update `BottomBorder` vars to be `BorderBottom`?

Comment (christiemolloy) [reply src/patternfly/components/ContextSelector/context-selector.scss]: updated @mcoker

**Comment by mattnolting** [comment]
How does this sound?

> The following are changes to variable names. All instances of the original variable should be removed as they are no longer used in PatternFly.

"The following are changes to variable names. Any reference to them should be changed or removed."

**Comment by mcoker** [comment]
I think if someone is referencing the old variable name, it's safe just to instruct them to reference the new one. I used this language in https://github.com/patternfly/patternfly/pull/2866

> Renames the following variables. Any references to the old vars will need to be updated to use the new vars.

**Comment by christiemolloy** [comment]
Updated @mcoker @mattnolting

**Comment by mattnolting** [comment]
wfm 👍

**Pull request merged**